examples/stats/prometheus/main.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"go.opencensus.io/exporter/stats/prometheus"
 	"go.opencensus.io/stats"
```

Comment:
Please revert the changes to the example. We want to keep the small and not want to demonstrate every possible aggregation.

Reply:
Roger that, I can do that. I guess I've perhaps send a follow up PR that uses the aggregations in tests.